rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/470_significant_texts.yml

```diff
@@ -185,23 +216,30 @@ setup:
       search:
         body:
           query:
-            match_all: {}
+            terms:
+              full_text:
+                - apple
+                - banana
+                - melon
+                - pear
           size: 0
           aggs:
             significant_texts:
               sampler:
-                shard_size: 7
+                shard_size: 200
```

Comment:
I think the intent here was to validate the the sampler aggregator actually limited input docs in an expected way.

Reply:
I wanted to make the test not depend on the result of the sampler because this is a test for the significant texts aggregator...anyway I think it does not make much difference in this case.

Reply:
The sampler was written for things like significant text aggregator. So their interaction is kind of important.